.env.example

```diff
@@ -1,2 +1,6 @@
 RACK_ENV=development
 REDISTOGO_URL=redis://localhost:6379/
+
```

Comment:
After register SP in Genius Center please store  APP_ID, APP_SECRET(32 bytes), GENIUS_AUTH_URL properly.